mod_reforged/hooks/config/strings.nut

```diff
@@ -967,6 +968,19 @@ foreach (vanillaDesc in vanillaDescriptions)
 			]
 		}]
  	}),
+	RF_Decisive = ::UPD.getDescription({
+		Fluff = "There is no time to wait!",
+		Effects = [{
+			Type = ::UPD.EffectType.Passive,
+			Description = [
+				"Gain a stack of decisiveness whenever you end your turn without having used \'Wait\', up to a maximum of 3.",
+				::MSU.Text.colorGreen("+15") + " Resolve while you have atleast 1 stack.",
+				::MSU.Text.colorGreen("-15%") + " Fatigue Cost for skills while you have atleast 2 stacks.",
+				::MSU.Text.colorGreen("+15%") + " Damage while you have atleast 3 stacks.",
```

Comment:
- `15% increased damage`
- The `at least` part in this line can be removed because 3 is the max.